src/Bedrock.Framework.Experimental/Bedrock.Framework.Experimental.csproj

```diff
@@ -18,6 +18,10 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Http.Connections.Client" Version="3.1.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Http.Connections.Client" Version="6.0.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Update="Nerdbank.GitVersioning" Version="3.4.244" />
```

Comment:
Why is this here?

Reply:
It is in the master too. I just updated it.